layouts/default.vue

```diff
@@ -26,8 +26,29 @@
       </v-list>
     </v-navigation-drawer>
     <v-toolbar fixed app :clipped-left="clipped">
-      <v-toolbar-side-icon @click="toggleDrawer"></v-toolbar-side-icon>
-      <v-img src="/icon.png" height="70%" contain position="left center" class="ml-2 mt-2"/>
+      <v-layout style="height: 100%; width: 100%; justify-content: space-between;" row align-center>
```

Comment:
Any reason to have the css in the template? How about putting it in `<style>`?

Reply:
put css in <style>, let me know what you think